### PR TITLE
Add NewMsgUpdateParams for host.

### PR DESCRIPTION
**Pull request by DimitrisJim**
## Description

Noticed when working on adding the e2e params test for host

closes: #XXXX

### Commit Message / Changelog Entry

```bash
chore: Add NewMsgUpdateParams for host.
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: modules/apps/27-interchain-accounts/host/types/msgs_test.go [hunk 1]
```diff
@@ -6,68 +6,47 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/types"
```
Comment (charleenfei) [inline modules/apps/27-interchain-accounts/host/types/msgs_test.go:7]: super duper nit -- group imports

File: modules/apps/27-interchain-accounts/host/types/msgs.go [hunk 2]
```diff
@@ -9,6 +9,14 @@ import (
 
 var _ sdk.Msg = (*MsgUpdateParams)(nil)
 
+// NewMsgUpdateParams creates a new MsgUpdateParams instance
```
Comment (charleenfei) [inline modules/apps/27-interchain-accounts/host/types/msgs.go:12]: i noticed there was another two instances where this util function could be used, in `transfer/types/msgs_test.go` and `interchainaccounts/controller/types/msgs_test.go`, but could be a followup and not blocker for this pr

**Review by charleenfei — APPROVED** [review]
just two nits but no blocker from me!

File: modules/apps/27-interchain-accounts/host/types/msgs_test.go (hunk 1 above)
Comment (DimitrisJim) [reply modules/apps/27-interchain-accounts/host/types/msgs_test.go:7]: oh damn got hit with the super duper variety

Comment (chatton) [reply modules/apps/27-interchain-accounts/host/types/msgs_test.go]: shouldn't the linter/formatter resolve this automatically?

**Review by chatton — APPROVED** [review]
LGTM! I like the simpler approach

File: modules/apps/27-interchain-accounts/host/types/msgs_test.go (hunk 1 above)
Comment (DimitrisJim) [reply modules/apps/27-interchain-accounts/host/types/msgs_test.go]: I want to say it doesn't separate by lines, it definitely has certain rules. Tbh haven't looked much into how the formatter for VSCode is configured for import paths.

**Review by colin-axner — APPROVED** [review]

**Pull request merged**